src/migrations/versions/cbbba858deb0_init.py

```diff
@@ -21,7 +20,9 @@ def upgrade() -> None:
         sa.Column("active_plugins", sa.ARRAY(sa.String()), nullable=True),
         sa.Column("name", sqlmodel.sql.sqltypes.AutoString(), nullable=False),
         sa.Column(
-            "ursadb_url", sqlmodel.sql.sqltypes.AutoString(), nullable=False
+            "ursadb_url",
+            sqlmodel.sql.sqltypes.AutoString(),
+            nullable=False,
```

Comment:
ideally this line should not change (this PR is completely unrelated to this file). Is there anything wong with the previous formatting?

Reply:
I had to modify this because black raise errors.